pkg/sessionctx/variable/sysvar.go

```diff
@@ -1054,6 +1054,20 @@ var defaultSysVars = []*SysVar{
 			return normalizedValue, nil
 		},
 	},
+	{
```

Comment:
I recommend incorporating the addition of the switch together with the main functionality. Adding a switch doesn't require many lines and won't affect the review process.